README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+I would say semantic HTML is more than just the way a page looks visually. There are links and containers and working functions. Not just color and style. 
```

Comment:
semantic html provides meaning to your html tags.  div doesn't carry meaning.  header does.  header is semantic.  div isn't